Program.cs

```diff
@@ -711,7 +772,7 @@ private void somethingCool()
             }
             else
             {
-                TypeWriter.WriteLine($"Hello {cottonUserName}");
+                TypeWriter.WriteLine($"Hello {cottonUserName} are you looking for ");
```

Comment:
This looks like it should not be here... looks like this line will write twice now.